public/index.js

```diff
@@ -149,10 +149,12 @@ function gabage_click(event) {
   InsectNetElement.style.position = "absolute";
   InsectNetElement.style.left = `${click_x}px`;
   InsectNetElement.style.top = `${click_y}px`;
-  InsectNetElement.style.width = 35 + "%";
-  InsectNetElement.style.position = "absolute";
+  InsectNetElement.style.width=120+"px";
   document.body.appendChild(InsectNetElement);
-  setTimeout(function () {
+  setTimeout(function(){
+    InsectNetElement.style.transform = "rotate(" + -90 + "deg)";
+  },100);
+  setTimeout(function(){
```

Comment:
```suggestion
  setTimeout(function () {
    InsectNetElement.style.transform = "rotate(-90deg)";
  },100);
  setTimeout(function () {
```

コードスタイルの修正
不要な文字列と数値の `+` 演算に見えたのですべて文字列でもいいように思いました